tgui/packages/tgui/webpack.config.js

```diff
@@ -36,8 +36,8 @@ module.exports = (env = {}, argv) => {
     },
     output: {
       path: argv.useTmpFolder
-        ? path.resolve(__dirname, './public/.tmp')
-        : path.resolve(__dirname, './public'),
+        ? path.resolve(__dirname, '../../../browserassets/tgui/.tmp')
```

Comment:
:kocmocpchela:

Reply:
I am unclear if this is a request for code change.